boskos/janitor/janitor.go

```diff
@@ -26,16 +28,41 @@ import (
 )
 
 var (
-	clean      = janitorClean
-	POOLSIZE   = 10 // Maximum concurrent janitor goroutines
-	BUFFERSIZE = 1  // Maximum holding resources
+	clean          = janitorClean
+	poolSize       = 10 // Maximum concurrent janitor goroutines
+	bufferSize     = 1  // Maximum holding resources
+	serviceAccount = flag.String("service-account", "", "Path to projects service account")
 )
 
+// exists returns whether the given file or directory exists or not
+func exists(path string) (bool, error) {
```

Comment:
Does anything use this function? If not can we delete it? Also is there a reason we need this over `_, err := os.Stat() && err == nil`?